MantidQt/CustomInterfaces/src/Indirect/ConvFit.cpp

```diff
@@ -222,6 +209,72 @@ void ConvFit::setup() {
   updatePlotOptions();
 }
 
+/** Setup FABADA minimizer options
+*
+*/
+void ConvFit::initFABADAOptions() {
```

Comment:
I realise this is how it was before but would it be possible to cut down how much this has to know about the actual properties on the minimizer?

If we used the `FuncMinimizerFactory` to create an instance of the `FABADA` minimizer we could then just iterate over the minimizer properties returned by `minimizer->getProperties()` to get the information. If new properties were added to the minimizer then they would automatically get pull over.

Would this be possible?